site/components/Navigation/NavItemMobile.tsx

```diff
@@ -8,34 +8,22 @@ interface Props extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
   name: string;
   id?: NavItemMobileID;
   url?: string;
-  toggledNavItemID?: string | undefined;
-  setToggledNavItemID?: (string: NavItemMobileID | undefined) => void;
   subMenu?: JSX.Element[];
   active?: boolean;
 }
 
 export const NavItemMobile: FC<Props> = (props) => {
-  const isToggled =
-    !!props.id &&
-    !!props.toggledNavItemID &&
-    props.id === props.toggledNavItemID;
-  const onToggleSubmenu = () => {
-    if (isToggled && props.setToggledNavItemID)
-      return props.setToggledNavItemID(undefined);
-    props.setToggledNavItemID &&
-      props.id &&
-      props.setToggledNavItemID(props.id);
-  };
+  const [isToggled, setIsToggled] = useState(false);
 
   return (
-    <div className={cx(styles.navMain_MobileItem)}>
+    <div className={cx(styles.navMain_MobileItem)} aria-expanded={isToggled}>
       <ButtonPrimary
         href={props.url}
         className={cx(styles.navMain_MobileLink, {
-          active: isToggled,
+          activeItem: isToggled,
         })}
         label={props.name}
-        onClick={onToggleSubmenu}
+        onClick={() => setIsToggled(!isToggled)}
```

Comment:
better: `onClick={() => setIsToggled(current => !current)}`
to read the previous state

Reply:
TY, noted for the future. Removed onClickfor now.